packages/app/src/messages/components/thread-details-messages.component.tsx

```diff
@@ -62,7 +62,20 @@ const ThreadDetailsMessages: FunctionComponent<Properties> = ({
     }
   }, [messages])
 
+  const isMessageIncomingDuringBeBottom = (): boolean => {
```

Comment:
Shouldn't the name be like `isMessageIncomingWhileScrollOnBottom`? A little bit longer, but seems to me it brings more sense and understanding.

Reply:
https://github.com/mudita/mudita-center/pull/753/commits/5b9679aa57536922aa290aa305ae6273a071021f